clang/lib/CConv/CheckedRegions.cpp

```diff
@@ -235,6 +237,44 @@ bool CheckedRegionFinder::VisitParmVarDecl(ParmVarDecl *PVD) {
   return true;
 }
 
+bool CheckedRegionFinder::isWild(std::set<ConstraintVariable*> &S) { 
+  for (auto Cv : S) 
+    if (Cv->hasWild(Info.getConstraints().getVariables()))
+      return true;
+
+  return false;
+}
+
+bool CheckedRegionFinder::isWild(std::set<FVConstraint*> *S) { 
+  for (auto Fv : *S) 
+    if (Fv->hasWild(Info.getConstraints().getVariables()))
+      return true;
+  return false;
+}
+
+bool CheckedRegionFinder::VisitDeclRefExpr(DeclRefExpr* DR) { 
+  auto T = DR->getType();
+  auto D = DR->getDecl();
+  auto var = Info.getVariable(D, Context);
+
+  bool IW = isWild(var) || isUncheckedPtr(T);
+
+
+  if (auto FD = dyn_cast<FunctionDecl>(D)) { 
+    auto FV = Info.getFuncConstraints(FD, Context);
+    IW |= isWild(FV);
```

Comment:
This is the kind of thing I was thinking about in my comment above: Just having a `IWild` variable that says whether you are wild or not, rather than counting.